wxt_cidrbot/cidrbot.py

```diff
@@ -51,11 +51,33 @@ def __init__(self):
         self.webex = webex_edit_message.webex_message()
         self.room_handle = cidrbot_room_setup.room_setup()
         self.roomID = ""
-        self.WEBEX_MESSAGE_CHAR_LIMIT = 4800
+        self.WEBEX_MESSAGE_CHAR_LIMIT = 4700
 
     def send_wbx_msg(self, room, message, pt_id):
         self.Api.messages.create(room, markdown=message, parentId=pt_id)
 
+    def send_wbx_messages(self, msgs_to_be_sent, room_id, message_id, pt_id, request_type):
+        """
+        sends all messages in a list and can either sent the message(s) as an edit or new reply
+
+        :param msgs_to_be_sent: list, the messages to be sent
+        :param room_id: string, the id for the webex room
+        :param message_id: string, id for the webex message bot will edit
+        :param pt_id: string, id of webex parent message bot responds to
+        :param request_type: string, whether request is daily update or to edit a message
+
+        return: bool, True if messages sent
+        """
+
+        if request_type == 'daily_message':
+            self.send_wbx_msg(room_id, msgs_to_be_sent.pop(0), pt_id)
+        elif request_type == 'edit_message':
+            self.webex.edit_message(message_id, msgs_to_be_sent.pop(0), room_id)
+        for msg in msgs_to_be_sent:
+            self.send_wbx_msg(room_id, msg, pt_id)
```

Comment:
I'm not sure I get how this works - the first message in a list will be either a daily message or an edit, but all the rest will be sent the same way?